common-libs/common/src/shared/seedTools.ts

```diff
@@ -130,6 +128,18 @@ export const convertDecryptedSeedBufferToString = (decryptedSeed: Buffer): strin
   return `${seedHex}++jolo`
 }
 
+export const processAnchoredElemDidSeed = (parsedSeed: ParseDecryptedSeedResult, did: string) => {
+  const { seed, externalKeys } = parsedSeed
+  const seedWithMethod = `${seed.toString('hex')}++${ELEM_ANCHORED_DID_METHOD}`
+  const additionalData = {
+    ...(externalKeys && { [EXTERNAL_KEYS_KEY]: externalKeys }),
+    [METADATA_KEY]: { anchoredDid: did },
```

Comment:
Please do not throw the existing metadata and other fields away:
```
const { seed, additionalData: originalAdditionalData } = parsedSeed
...
const additionalData = {
  ...originalAdditionalData,
  [METADATA_KEY]: {
    ...originalAdditionalData?.[METADATA_KEY],
    anchoredDid: did,
  },
}
```

Right now we don't have any other fields in additional data, and metadata does not have any other keys, but this might change in the future, and this method should add anchored did to the other data, not replace all the other data with anchored did and external keys.